### PR TITLE
S01 us10addcutomer

### PR DESCRIPTION
**Pull request by plbonnier**

### CODE REVIEW

**Review by nickyrasa — CHANGES_REQUESTED** [review]
il faut faire la fonction show car si on ajoute un client, rien ne s'affiche

**Review by Guadanino — APPROVED** [review]

**Review by nickyrasa — APPROVED** [review]

File: src/Controller/CategoryController.php [hunk 1]
```diff
@@ -6,9 +6,12 @@
 
 class CategoryController extends AbstractController
 {
-    public function index(): string
+    public function indexCategory(): string
```
Comment (benrcd) [inline src/Controller/CategoryController.php:9]: showCategories() would be a better fitting name.

File: src/Controller/CustomerController.php [hunk 2]
```diff
@@ -3,11 +3,67 @@
...
+use App\Model\TypeManager;
 
 class CustomerController extends AbstractController
 {
-    public function index(): string
+    public function indexCustomer(): string
     {
-        return $this->twig->render('Customer/index.html.twig');
+        $customerManager = new CustomerManager();
+        $customers = $customerManager->getAll();
```
Comment (benrcd) [inline src/Controller/CustomerController.php]: getAllCustomer would be a better name.

**Review by benrcd — CHANGES_REQUESTED** [review]
Complete the form security

**Review by benrcd — APPROVED** [review]

**Pull request merged**